mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -90,6 +108,12 @@ private[regression] trait AFTSurvivalRegressionParams extends Params
       SchemaUtils.checkColumnType(schema, $(censorCol), DoubleType)
       SchemaUtils.checkColumnType(schema, $(labelCol), DoubleType)
     }
+    if (hasQuantileProbabilities && hasQuantilesCol) {
+      SchemaUtils.appendColumn(schema, $(quantilesCol), new VectorUDT)
+    } else if (hasQuantileProbabilities || hasQuantilesCol) {
+      logWarning("The output will not include quantiles column. " +
+        "If you want to include, please set both quantileProbabilities and quantilesCol.")
+    }
```

Comment:
The semantic:
1, Users set both ```quantileProbabilities``` and ```quantilesCol``` => output w/ quantiles column
2, Users set ```quantileProbabilities``` but not ```quantilesCol``` => print warning here, output w/o quantiles column and users can use ```predictQuantiles```
3, Users set ```quantilesCol``` but not ```quantileProbabilities``` => print warning here and throw ```IllegalArgumentException``` at [L284](https://github.com/yanboliang/spark/blob/spark-10686/mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala#L284)
4, Neither of ```quantileProbabilities``` and ```quantilesCol``` is set =>  Users purpose to output w/o quantiles column